tests/benchmarks/test_join.py

```diff
@@ -1,52 +1,51 @@
+import dask
 import dask.dataframe as dd
 import pytest
 
 from ..utils_test import cluster_memory, run_up_to_nthreads, timeseries_of_size
 
-mem_mult = [
-    0.1,
-    pytest.param(
-        1,
-        marks=pytest.mark.skip(reason="Does not finish"),
-    ),
-    pytest.param(
-        10,
-        marks=pytest.mark.skip(reason="Does not finish"),
-    ),
+params = [
+    (0.1, "tasks"),
+    # shuffling takes a long time with 1 or higher
+    (0.1, "p2p"),
+    # (1, "p2p"),
```

Comment:
#633 should allow us to enable this in a follow-up

Reply:
Shouldn't we mark them then as `marks=pytest.mark.skip(reason="Does not finish, fix when #633 is merged")`

If not, open an issue to state that when the #633 is merged we need to uncomment this